src/project/projecttypes.h

```diff
@@ -88,6 +88,35 @@ struct CloudProjectInfo {
     }
 };
 
+struct ExportInfo {
+    QString id;
+    io::path_t path;
+
+    ExportInfo() {}
```

Comment:
Is there any reason to not write `ExportInfo() = default;`? And why are the copy constructor and `operator=(…)` defined explicitly?

Reply:
I was getting a bunch of warnings and errors at first. So when I added them, it solved them. But now it works fine without those functions